scripts/generate-videos/main.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+#  Copyright © 2015-2019 Cask Data, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import argparse
+import os
+import json
+from youtube import API
+
+MAX_RESULTS = 50
```

Comment:
this will not fetch all right? Should we increase this to 999, and make it configurable?

Reply:
We fetch all videos, this is just the constant used for request since default value is `5` and `50` is maximum allowed page size to be set for a API request.